scipy/signal/filter_design.py

```diff
@@ -248,6 +248,104 @@ def freqz(b, a=1, worN=None, whole=0, plot=None):
     return w, h
 
 
+def group_delay(system, w=None, whole=False):
+    r"""Compute the group delay of a digital filter.
+
+    The group delay measures by how many samples amplitude envelopes of
+    various spectral components of a signal are delayed by a filter.
+    It is formally defined as the derivative of continuous (unwrapped) phase::
+
+               d        jw
+     D(w) = - -- arg H(e)
+              dw
+
+    Parameters
+    ----------
+    system : tuple of array_like (b, a)
+        Numerator and denominator coefficients of a filter transfer function.
+    w : {None, int, array-like}, optional
+        If None (default), then compute at 512 frequencies equally spaced
+        around the unit circle.
+        If a single integer, then compute at that many frequencies.
+        If array, compute the delay at the frequencies given
+        (in radians/sample).
+    whole : bool, optional
+        Normally, frequencies are computed from 0 to the Nyquist frequency,
+        pi radians/sample (upper-half of unit-circle).  If `whole` is True,
+        compute frequencies from 0 to `2*pi` radians/sample.
```

Comment:
2*pi this should be in double ticks, right?  single ticks are for variables, double are for code?